python/arcticdb/version_store/_normalization.py

```diff
@@ -234,8 +234,6 @@ def _to_primitive(arr, arr_name, dynamic_strings, string_max_len=None, coerce_co
             casted_arr = coerce_string_column_to_fixed_length_array(arr, type(sample), string_max_len)
     elif dynamic_strings and sample is None:  # arr is entirely empty
         return arr
-    elif isinstance(sample, bool) or isinstance(sample, np.ndarray):
```

Comment:
Won't this lead to problems when we allow writing nullable bools or np arrays in dataframes? Is there a feature flag for writing those types?

Reply:
We currently don't have a feature flag for the nullable booleans. We have this line here and [this](https://github.com/man-group/ArcticDB/blob/2cad4622122e730032271a0e936a8c590f611cd4/cpp/arcticdb/python/python_to_tensor_frame.cpp#L52). While it's possible to add a flag this normalization function is a bit deep in the call chain. IMO it would be a fair amount of work adding the feature flag and (and later on removing it) compared to adding this PR and reverting it.